dotnet/src/Connectors/Connectors.GoogleVertexAI/Core/Gemini/GeminiContent.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace Microsoft.SemanticKernel.Connectors.GoogleVertexAI.Core;
+
+/// <summary>
+/// The base structured datatype containing multi-part content of a message.
+/// </summary>
+internal sealed class GeminiContent
+{
+    /// <summary>
+    /// Ordered Parts that constitute a single message. Parts may have different MIME types.
+    /// </summary>
+    [JsonPropertyName("parts")]
+    public IList<GeminiPart>? Parts { get; set; }
+
+    /// <summary>
+    /// Optional. The producer of the content. Must be either 'user' or 'model'.
```

Comment:
nit: Looking at `AuthorRoleConverter` class, it also handles `function` value, not sure if we need to update it here as well.

Reply:
done